### PR TITLE
Increase retention for activity logs that we don't keep forever

### LINKED ISSUE #20228: Increase retention for activity logs that we don't keep forever
We want to increase the retention for activity logs that we don't keep forever to 1 year.

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #20228.

### CODE REVIEW

File: src/olympia/amo/cron.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def days_ago(days):
     log.info('Collecting data to delete')
 
     logs = (
-        ActivityLog.objects.filter(created__lt=days_ago(180))
+        ActivityLog.objects.filter(created__lt=days_ago(365))
```
Comment (eviljeff) [inline src/olympia/amo/cron.py:36]: Can we have this as a constant defined somewhere?

File: src/olympia/amo/tests/test_cron.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def test_stale_activity_logs_deletion(self, storage_mock):
             ActivityLog.objects.create(action=amo.LOG.THROTTLED.id, user=user),
             # Old activity from action that should always be kept.
             ActivityLog.objects.create(
-                action=amo.LOG.LOG_IN.id, user=user, created=self.days_ago(181)
+                action=amo.LOG.LOG_IN.id, user=user, created=self.days_ago(366)
```
Comment (eviljeff) [inline src/olympia/amo/tests/test_cron.py:34]: And then `constant + 1` here and the other test.

**Review by eviljeff — APPROVED** [review]

File: src/olympia/amo/tests/test_cron.py (hunk 2 above)
Comment (wagnerand) [reply src/olympia/amo/tests/test_cron.py]: [`3ea0d55` (#20229)](https://github.com/mozilla/addons-server/pull/20229/commits/3ea0d55d748cfa225e4f4d532e5751faa0650bca)

File: src/olympia/amo/cron.py (hunk 1 above)
Comment (wagnerand) [reply src/olympia/amo/cron.py]: [`3ea0d55` (#20229)](https://github.com/mozilla/addons-server/pull/20229/commits/3ea0d55d748cfa225e4f4d532e5751faa0650bca)

**Pull request merged**